README.md

```diff
@@ -654,6 +654,74 @@ instance.delete()
 
 **Do not use delete() in production!**
 
+Bulk-updating sandboxes
+-----------------------
+
+OpenCraft IM provides a tagging mechanism that you can use to update instances in bulk.
+The following script shows how to update multiple instances at once from the Django shell:
+
+```python
+# Author: Brandon DeRosier <bdero@opencraft.com>
```

Comment:
@antoviaque Ok, well for our next iteration, I suggest we put steps 3-7 into an API within the codebase and to think about how to make them a bit more generic to handle cases like migrations.

For the rest, just because we have to type a few commands into the shell doesn't mean it's not "automated" - it's the same situation as provisioning a new instance right now, which requires typing some commands into a shell. The shell is the UI, and 99% of the work is still automated. Same here - because of the complexity of the types of upgrades, migrations, changes, deployments, etc. that we may need to do, it makes sense to me to keep the shell as the UI for such things, even though 99% of the work is automated within the codebase. Later on we can create another UI that's actually HTML-based if we see enough common patterns that fit that workflow.